docker-compose.yml

```diff
@@ -11,6 +11,8 @@ services:
     environment:
       - LAB_RESULTS_LOGGING_LEVEL=INFO
       - LAB_RESULTS_OUTBOUND_SERVER_PORT=8080
+      - NHAIS_SSL_TRUST_STORE_URL=
+      - NHAIS_SSL_TRUST_STORE_PASSWORD=
```

Comment:
Shouldn't this be

```
  - NHAIS_SSL_TRUST_STORE_URL=${NHAIS_SSL_TRUST_STORE_URL}
  - NHAIS_SSL_TRUST_STORE_PASSWORD=${NHAIS_SSL_TRUST_STORE_PASSWORD}
```

?

Reply:
not for int tests

Reply:
doesn't have to
docker-compose is used for dockerized test only and this will work without custom truststore